src/Controls/src/Core/BindableProperty.cs

```diff
@@ -38,13 +38,6 @@ public sealed class BindableProperty
 
 		public delegate bool ValidateValueDelegate<in TPropertyType>(BindableObject bindable, TPropertyType value);
 
-		static readonly Dictionary<Type, TypeConverter> KnownTypeConverters = new Dictionary<Type, TypeConverter>
-		{
-			{ typeof(Uri), new UriTypeConverter() },
-			{ typeof(Color), new ColorTypeConverter() },
-			{ typeof(Easing), new EasingTypeConverter() },
```

Comment:
The issue with `Uri` and `Easing` is that the converter isn't marked on the Type.

They are both defined in a library that can't reference the library the converter is in.

We could fix this in `Easing` using a string reference instead. But that isn't very safe nor performant since you need to use `Type.GetType` and hope the Type is around. Maybe we could move `EasingTypeConverter` from `Microsoft.Maui.Controls` to `Microsoft.Maui` to solve this?

But Uri will have to be solved a different way. Going through System.ComponentModel.TypeDescriptor.GetConverter, it does have this mapping here:

https://github.com/dotnet/runtime/blob/4ed3063d8bc434fd29fe63cf093c17baa9ff8fd3/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs#L180

Reply:
you're right about UriTypeConverter, I've been too quick to clean stuffs. what worries me is that we didn't got any failing test :(

Reply:
EasingTypeConverter can be moved, easily, I'm doing it. We definitely should replace our build-in conversion logic for native types with TypeDescriptor, let's track that on #5660

Reply:
> what worries me is that we didn't got any failing test 

💯